np6.py

```diff
@@ -0,0 +1,18 @@
+import matplotlib.pyplot as mplot
+import numpy as np
+
+
+def plot_sin(x):
```

Comment:
Use type hints and return types -docum
https://www.geeksforgeeks.org/duck-typing-in-python/
https://stackoverflow.com/questions/125367/dynamic-type-languages-versus-static-type-languages
https://docs.python.org/3/library/typing.html
https://realpython.com/lessons/pros-and-cons-type-hints/